etc/inc/disable-programs.inc

```diff
@@ -466,6 +467,7 @@ blacklist ${HOME}/.config/iridium
 blacklist ${HOME}/.config/itch
 blacklist ${HOME}/.config/jami
 blacklist ${HOME}/.config/jd-gui.cfg
+blacklist ${HOME}/.config/jgit
```

Comment:
AFAIK jgit is the generic git implementation used by Java programs, so this
could possibly affect programs like Eclipse and SonarQube.